samples/v2/sample_test.py

```diff
@@ -20,16 +20,19 @@
 from typing import List
 import unittest
 
+import collected_parameters
 import component_with_optional_inputs
 import hello_world
 import kfp
 from kfp.dsl.graph_component import GraphComponent
+from modelcar import modelcar
```

Comment:
Could you please fix the imports? I think as part of a rebase, this PR added duplicate imports for `modelcar` and `pipeline_with_placeholders`.